DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -86,6 +86,9 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->setParameter($this->getAlias().'.compiler_pass.tag_annotations', $config['tags']['enabled']);
         if ($config['tags']['enabled']) {
+            // only load tag_invalidate if tags are enabled
+            $loader->load('tag_invalidate.xml');
```

Comment:
I also think it would be better to put all the "tag related stuff" in one file.
But as it's not my bundle... ;)